.eslintrc.js

```diff
@@ -17,7 +17,8 @@ module.exports = {
   'env': {
     'node': true,
   },
-  'plugins': ['node', 'prettier', 'header'],
+  parser: '@typescript-eslint/parser',
+  'plugins': ['node', 'prettier', 'header', '@typescript-eslint'],
   'extends': [
     'eslint:recommended',
```

Comment:
I'm not sure whether this adds anything in our case, or we do not need it at all but [in this documentation](https://github.com/typescript-eslint/typescript-eslint/blob/master/docs/getting-started/linting/README.md#configuration) `'plugin:@typescript-eslint/recommended'` took place in the `'extends'` array:
```suggestion
    'eslint:recommended',
    'plugin:@typescript-eslint/recommended',
```

@bessbd wdyt? Do you see any benefit from it?

Reply:
The idea sounds good and I initially had it there, but I didn't really see the value: https://gist.github.com/bessbd/f13b492deeff5e72a0e63c517fceeb04 . It looked much more like noise to me.
Lmk if you'd be okay with adding that later or please help me understand what we're missing by not adding that `extends` right now.

Reply:
If you did not see the value in it, I trust you, I was just curious. I took a glance at the gist you have shared but I did not see anything that would be beneficial for us.